scripts/plotting/tem.py

```diff
@@ -107,20 +112,27 @@ def tem(adf):
     else:
         var_list = ['uzm','epfy','epfz','vtem','wtem','psitem','utendepfd']
 
-    #Baseline TEM location
-    input_loc_idx = Path(tem_loc) / base_name
-
     #Check if comparing against obs
-    if obs:
+    if adf.compare_obs:
+        obs = True
         #Set TEM file for observations
-        base_file_name = f'{base_name}.TEMdiag.nc'
+        base_file_name = 'Obs.TEMdiag.nc'
     else:
-        #Set TEM file for baseline
-        base_file_name = f'{base_name}.TEMdiag_{syear_baseline}-{eyear_baseline}.nc'
+        obs = False
+        base_name = adf.get_baseline_info("cam_case_name", required=True)
```

Comment:
I might put these lines underneath the `else` block of the `if tem_base_loc is None` check below.

Reply:
I left the `base_name` line so that the it's available in the print statement for the 
```python
if tem_base_loc is None
```
But I moved the `obs = False` to the `else` block